network-generator/src/main/kotlin/dev/icerock/moko/network/KtorCodegen.kt

```diff
@@ -47,10 +48,12 @@ class KtorCodegen : AbstractKotlinCodegen() {
         typeMapping["UUID"] = "kotlin.String"
         typeMapping["URI"] = "kotlin.String"
         typeMapping["object"] = "JsonObject"
+        typeMapping["decimal"] = "BigNum"
```

Comment:
for "number" should be too as i  know

Reply:
I don't know why, but it already works for number 
here example where `amount` has format `number`, and `total` - `decimal`
```
/**
 * 
 * @param amount 
 * @param total 
 */
@Serializable
public data class DecimalTest (
    @Serializable(with = BigNumSerializer::class) 
    @SerialName("amount")
    val amount: com.soywiz.kbignum.BigNum? = null,
    @Serializable(with = BigNumSerializer::class) 
    @SerialName("total")
    val total: com.soywiz.kbignum.BigNum? = null

) 
```